README.md

```diff
@@ -18,18 +18,21 @@ Easy management of products, rentals and customers
 
 ### Setup
 
-1. In `docker-compose.yml`, add the path to your local excel file and insert your WooCommerce key and secret
-2. Run `docker-compose up` and wait for the excel_to_couchdb container to terminate
-3. Run `cd Frontend && npm install`
+- Run `cd Frontend && npm install`
+- For running the application other than in the demo mode, proceed additionally with 
+	1. In `docker-compose.yml`, add the path to your local excel file (replace `<LOCAL_EXCEL_FILE>` with the absolute path) and insert your WooCommerce key and secret
+	2. Run `docker-compose up` and wait for the excel_to_couchdb container to terminate
 
 ### Run
 
-    cd Frontend && npm run build && npm run start
-
-or
+Just the demo
 
     cd Frontend && npm run demo
 
-or
+or the real build
+
+    cd Frontend && npm run build && npm run start
+
+or the dev build
 
     cd Frontend && npm run dev
```

Comment:
This is the same build as "npm run start", it just rebuilds automatically when files are changed